src/Content/Conversation.php

```diff
@@ -287,15 +287,15 @@ public function statusEditor(array $x = [], $notes_cid = 0, $popup = false)
 		$this->profiler->startRecording('rendering');
 		$o = '';
 
-		$x['allow_location']   = $x['allow_location']   ?? $user['allow_location'];
+		$x['allow_location']   = $x['allow_location'] ?? $user['allow_location'];
 		$x['default_location'] = $x['default_location'] ?? $user['default-location'];
-		$x['nickname']         = $x['nickname']         ?? $user['nickname'];
-		$x['lockstate']        = $x['lockstate']        ?? ACL::getLockstateForUserId($user['uid']) ? 'lock' : 'unlock';
-		$x['acl']              = $x['acl']              ?? ACL::getFullSelectorHTML($this->page, $user['uid'], true);
-		$x['bang']             = $x['bang']             ?? '';
-		$x['visitor']          = $x['visitor']          ?? 'block';
-		$x['is_owner']         = $x['is_owner']         ?? true;
-		$x['profile_uid']      = $x['profile_uid']      ?? local_user();
+		$x['nickname']         = $x['nickname'] ?? $user['nickname'];
+		$x['lockstate']        = $x['lockstate'] ?? ACL::getLockstateForUserId($user['uid']) ? 'lock' : 'unlock';
+		$x['acl']              = $x['acl'] ?? ACL::getFullSelectorHTML($this->page, $user['uid'], true);
+		$x['bang']             = $x['bang'] ?? '';
+		$x['visitor']          = $x['visitor'] ?? 'block';
+		$x['is_owner']         = $x['is_owner'] ?? true;
+		$x['profile_uid']      = $x['profile_uid'] ?? local_user();
```

Comment:
Nooo, my nice alignment!

Reply:
Sorry. I wanted to keep it. But the system insisted on removing it.

Reply:
"the system" as if we couldn't change it.

Although I'm pretty sure there isn't a setting in `phpcs` to make these specific alignments.

Reply:
If you can make these alignments then I'm happy reverting these changes.

Reply:
Thank, I had a quick look at the PHPCS rules and none are related to ternary operator alignment.

Reply:
The check tells:
```
src/src/Content/Conversation.php (ternary_operator_spaces, binary_operator_spaces)
```
I guess that `ternary_operator_spaces` and `binary_operator_spaces` are the rules?